Mathlib/MeasureTheory/Group/FundamentalDomain.lean

```diff
@@ -253,6 +253,10 @@ theorem lintegral_eq_tsum' (h : IsFundamentalDomain G s μ) (f : α → ℝ≥0
 #align measure_theory.is_fundamental_domain.lintegral_eq_tsum' MeasureTheory.IsFundamentalDomain.lintegral_eq_tsum'
 #align measure_theory.is_add_fundamental_domain.lintegral_eq_tsum' MeasureTheory.IsAddFundamentalDomain.lintegral_eq_tsum'
 
+@[to_additive] lemma lintegral_eq_tsum'' (h : IsFundamentalDomain G s μ) (f : α → ℝ≥0∞) :
+  ∫⁻ x, f x ∂μ = ∑' g : G, ∫⁻ x in s, f (g • x) ∂μ :=
+(lintegral_eq_tsum' h f).trans ((Equiv.inv G).tsum_eq (fun g ↦ ∫⁻ (x : α) in s, f (g • x) ∂μ))
```

Comment:
indentation is wrong here

Reply:
sorry can you please tell me what's wrong? I don't see. Thanks!

Reply:
Ok I think I've got it.

Reply:
The indentation level of the type (aka statement) has changed with the switch to Lean 4.
The convention is now to indent by 4 spaces on a new line after the `:`.

Reply:
If the statement is short, you can of course still write it on the same line as the `:` with only 1 space in between.